fdbclient/FDBTypes.h

```diff
@@ -74,6 +74,8 @@ struct Tag {
 
 	int toTagDataIndex() const { return locality >= 0 ? 2 * locality : 1 - (2 * locality); }
 
+	bool isPseudoTag() const { return locality < 0; }
```

Comment:
Maybe `isNonPrimaryTLogType()` per [doc](https://quip-apple.com/shipAEZOruVX)  and `FBTypes.h`

"When it is negative value, it decides which types of tags the tag belongs to. For example, if it is -2, it is log router tag, and its id is used to decide which log router the tagged mutation should be sent to. "

Reply:
Ok, done.